_posts/2023-07-28-ajaxmanager.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "[CVE-2023-33493] Unrestricted Upload of File with Dangerous Type in the Ajaxmanager File and Database explorer (ajaxmanager) module from RSI for PrestaShop"
+categories: module
+author:
+- Profileo.com
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,ajaxmanager"
+severity: "critical (10)"
+---
+
+An "Unrestricted Upload of File with Dangerous Type" vulnerability exists in the Ajaxmanager File and Database explorer (ajaxmanager) module, from RSI, for PrestaShop, for all versions (including latest version 2.3.0). This allows remote attackers to upload dangerous files without restriction.
```

Comment:
An "Unrestricted Upload of File with Dangerous Type" vulnerability exists in the Ajaxmanager File and Database explorer (ajaxmanager) module, from RSI, for PrestaShop, in all versions (including the latest version 2.3.0). This allows remote attackers to upload dangerous files without restriction.